icons/xd/xd-line.svg

```diff
@@ -0,0 +1,15 @@
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 128 128" style="enable-background:new 0 0 128 128;" xml:space="preserve">
```

Comment:
`enable-background` is [depreceated](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/enable-background) and should be removed. `x` and `y` is not required and can be removed as well.

```suggestion
<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"
	 viewBox="0 0 128 128"  xml:space="preserve">
```